### PR TITLE
Add Mamba H743 v4

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-mambaH743v4.rst [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+- CRSF also requires a TX1 connection, in addition to RX1, and automatically provides telemetry. Set :ref:`SERIAL1_OPTIONS<SERIAL7_OPTIONS>` to "0".
+
+- SRXL2 requires a connection to TX1 and automatically provides telemetry.  Set :ref:`SERIAL1_OPTIONS<SERIAL1_OPTIONS>` to "4".
+
+Any UART can be used for RC system connections in ArduPilot also, and is compatible with all protocols except PPM. See :ref:`common-rc-systems` for details.
+
+FrSky Telemetry
+===============
+
+FrSky Telemetry is supported any UART TX pin. You need to set the following parameters to enable support for FrSky S.PORT (example given for UART5)
```
Comment (andyp1per) [inline common/source/docs/common-mambaH743v4.rst:90]: supported on

File: common/source/docs/common-mambaH743v4.rst [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+-   PWM 5,6 in group2
+-   PWM 7,8 in group3
+-   PWM 9 in group4
+
+Channels within the same group need to use the same output rate. If any channel in a group uses DShot then all channels in the group need to use DShot. Channels 1-8 support bi-directional DShot.
+
+Battery Monitoring
+==================
+
+The board does not a built-in current sensor. The voltage sensor can handle up to 6S LiPo batteries.
```
Comment (andyp1per) [inline common/source/docs/common-mambaH743v4.rst:117]: does not have

**Review by andyp1per — APPROVED** [review]
Just a couple of typos

File: common/source/docs/common-mambaH743v4.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-mambaH743v4.rst]: fixed

File: common/source/docs/common-mambaH743v4.rst (hunk 2 above)
Comment (Hwurzburg) [reply common/source/docs/common-mambaH743v4.rst]: fixed

**Pull request merged**